src/main/java/pl/betoncraft/betonquest/compatibility/Compatibility.java

```diff
@@ -37,6 +37,7 @@
 import pl.betoncraft.betonquest.compatibility.betonlangapi.BetonLangAPIEvent;
 import pl.betoncraft.betonquest.compatibility.betonlangapi.LangChangeListener;
 import pl.betoncraft.betonquest.compatibility.bountifulapi.BountifulTitleEvent;
+import pl.betoncraft.betonquest.compatibility.brewery.*;
```

Comment:
Can you expand this to individual imports? It will be consistent that way.

Reply:
Fine, although I gotta say that seems like unnecessary lines to me - literally the whole package is imported.

Reply:
It's fine as long as you're using an IDE. But when looking at the code without "jump to declaration" functionality, it suddenly becomes hard to know what symbol exactly you're looking at. With explicit imports you can always find where each class is located and navigate there manually.

I hate it when looking at someone's code on GitHub and can't tell where does each class come from.